inlang/source-code/plugins/inlang-message-format/src/storageSchema.ts

```diff
@@ -4,7 +4,7 @@ const InternalProperties = Type.Object(
 	{
 		$schema: Type.Optional(Type.Literal("https://inlang.com/schema/inlang-message-format")),
 	},
-	{ additionalProperties: false }
+	{ additionalProperties: true }
```

Comment:
I was unsure if I should remove the whole object or just the `additionalProperties` property leaving `{ }`, so it was easier to change the value to `true`.